src/parser/xmlHandler.js

```diff
@@ -249,7 +249,12 @@ class XMLHandler {
   }
 
   addAttributes(node, nsContext, descriptor, val, attrs) {
-    var elements = {}, attributes = {};
+    var attrDescriptors = (descriptor && descriptor.attributes) || [];
```

Comment:
I may not have the knowledge to review the feature change in this repo, just the old code declares `element` as `{}` but the new code doesn't, if this is an intended change then never mind :)